src/phonic.ts

```diff
@@ -81,4 +88,11 @@ export class Phonic {
   async get<T>(path: string) {
     return this.fetchRequest<T>(path, { method: "GET" });
   }
+
+  async post<T>(path: string, body: PostBody) {
```

Comment:
I feel that listing all the possible PostBody values (which is only going to grow over time) is not going to provide a lot of value to the user of the SDK, and if we properly test the methods we should be covered. I suggest we type the body here as `Record<string, unknown>`.